### PR TITLE
Birthplace/Foreign Realestate updates

### LINKED ISSUE #1328: Foreign activities - Foreign real estate - address block
Provide the location/address of property:

Should only have:
- [x] Street
- [x] City
- [x] Country

screen shot 2017-05-30 at 2 53 35 pm

screen shot 2017-05-30 at 2 53 50 pm

### PR DESCRIPTION
**Pull request by floresj**
Resolves #1328

### CODE REVIEW

File: src/validators/birthplace.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ export default class BirthPlaceValidator {
   constructor (state = {}, props = {}) {
     this.domestic = state.domestic
     this.country = state.country
+    this.street = state.street
     this.city = state.city
     this.state = state.state
     this.county = state.county
-    this.hideCounty = (props || {}).hideCounty || false
+    this.hideCounty = [true, false].includes((props || {}).hideCounty) ? props.hideCounty : false
+    this.hideStreet = [true, false].includes((props || {}).hideStreet) ? props.hideStreet : true
```
Comment (bmallred) [inline src/validators/birthplace.js]: Nice

**Comment by bmallred** [comment]
place-with-street

**Comment by adelevie** [comment]
Looking good so far.

**Comment by floresj** [comment]
Updates have been moved to PR #1477.

**Pull request closed without merging**